dashboard/app/assets/stylesheets/application.scss

```diff
@@ -219,19 +219,19 @@ img.video_thumbnail {
     text-overflow: ellipsis;
   }
 
-  @media screen and (min-width: 1011px) {
+  @media screen and (min-width: 1031px) {
```

Comment:
I'm not a fan of this one because it takes us past the crucial `1024px`, which has been a very common screen width, and means a bunch of people who could previously see the links across the header won't see them any more.

Reply:
Yep, agreed –– the wrapping is happening right around the 1027px mark, which is beyond the 1024. @dmcavoy I think this means we go with the "Course Curriculum" solution for now

Reply:
Alright thats fine. Lets do that